js/script.js

```diff
@@ -0,0 +1,787 @@
+  {
+    name: "floor",
+    symbol: "floor",
+    formula: "Math.floor(",
+    type: "math_function",
+  },
+  {
+    name: "ceil",
+    symbol: "ceil",
+    formula: "Math.ceil(",
+    type: "math_function",
+  },
+  {
+    name: "round",
+    symbol: "round",
+    formula: "Math.round(",
+    type: "math_function",
+  },
+  {
+    name: "sign",
+    symbol: "sign",
+    formula: "Math.sign(",
+    type: "math_function",
+  },
+  {
+    name: "fixedToExponent",
+    symbol: "F-E",
+    formula: "toExponent(",
+    type: "math_function",
+  },
+  {
+    name: "ANS",
+    symbol: "ANS",
+    formula: "ans",
+    type: "number",
+  },
+  {
+    name: "percent",
+    symbol: "%",
+    formula: "/100",
+    type: "number",
+  },
+  {
+    name: "comma",
+    symbol: ".",
+    formula: ".",
+    type: "number",
+  },
+  {
+    name: "0",
+    symbol: 0,
+    formula: 0,
+    type: "number",
+  },
+  {
+    name: "calculate",
+    symbol: "=",
+    formula: "=",
+    type: "calculate",
+  },
+  {
+    name: "addition",
+    symbol: "+",
+    formula: "+",
+    type: "operator",
+  },
+  {
+    name: "Inverse",
+    symbol: "1/x",
+    formula: "(1/",
+    type: "number",
+  },
+  {
+    name: "mod",
+    symbol: "mod",
+    formula: "%",
+    type: "operator",
+  },
+  {
+    name: "value-mod",
+    symbol: "|x|",
+    formula: "Math.abs(",
+    type: "operator",
+  },
+  {
+    name: "memory-store",
+    symbol: "MS",
+    formula: false,
+    type: "memory_function",
+  },
+  {
+    name: "memory-clear",
+    symbol: "MC",
+    formula: false,
+    type: "memory_function",
+  },
+  {
+    name: "memory-recall",
+    symbol: "MR",
+    formula: false,
+    type: "memory_function",
+  },
+  {
+    name: "memory-add",
+    symbol: "M+",
+    formula: false,
+    type: "memory_function",
+  },
+  {
+    name: "memory-subtract",
+    symbol: "M-",
+    formula: false,
+    type: "memory_function",
+  },
+];
+const input_element = document.querySelector(".btn-wrap");
+const output_operation_element = document.querySelector("#display");
+let RADIAN = true;
+let memory_value = 0;
+const deg_btn = document.getElementById("deg");
+
+trig_arrow.addEventListener("click", (event) => {
+  event.stopPropagation();
+});
+
+trig_block.addEventListener("click", (event) => {
+  event.preventDefault();
+  if (trig_container.style.display == "block") {
+    trig_container.style.display = "none";
+    trig_arrow.style.transform = "rotate(90deg)";
+  } else {
+    trig_container.style.display = "block";
+    trig_arrow.style.transform = "rotate(270deg)";
+  }
+});
+
+f_block.addEventListener("click", () => {
+  if (func_container.style.display == "block") {
+    func_container.style.display = "none";
+    func_arrow.style.transform = "rotate(90deg)";
+  } else {
+    func_container.style.display = "block";
+    func_arrow.style.transform = "rotate(270deg)";
+  }
+});
+
+function angleToggle() {
+  deg_btn.classList.toggle("active-angle");
+  event.stopPropagation();
```

Comment:
If this **event** is referring to **window.event**, please remove it.
window.event is deprecated and it will be undefined at runtime.